### PR TITLE
Polish details in right sidebar

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Description
Details accordion needs some more polishing

Additional: Update codeowners, unify data-test(-)ids, remove dead code

## Related Issue
- Works towards closing #5466

### CODE REVIEW

**Review by kulmann — APPROVED** [review]

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17184/49/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

File: packages/web-app-files/src/components/SideBar/Details/FileDetails.vue [hunk 1]
```diff
@@ -1,14 +1,14 @@
 <template>
   <div id="oc-file-details-sidebar">
     <oc-loader v-if="loading" />
-    <div v-else>
+    <div v-if="!loading && hasContent">
```
Comment (kulmann) [inline packages/web-app-files/src/components/SideBar/Details/FileDetails.vue:4]:
```suggestion
    <div v-else-if="hasContent">
```

**Review by kulmann — APPROVED** [review]
Tiny chance for improvement in the comments. Other than that: LGTM 🚀  nice improvements, thank you! 💪

File: packages/web-app-files/src/components/SideBar/Details/FileDetails.vue (hunk 1 above)
Comment (pascalwengerter) [reply packages/web-app-files/src/components/SideBar/Details/FileDetails.vue:4]: Adding this and bumping ODS to `v8.0.0` once it's released, then re-running CI and will merge this once it's green!

**Review by kulmann — APPROVED** [review]
LGTM 👍

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17186/49/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Pull request merged**